src/controllers/trust.individual.beneficial.owner.controller.ts

```diff
@@ -0,0 +1,71 @@
+import { NextFunction, Request, Response } from 'express';
+import * as config from '../config';
+import { logger } from '../utils/logger';
+import { safeRedirect } from '../utils/http.ext';
+import { getApplicationData } from '../utils/application.data';
+import * as CommonTrustDataMapper from '../utils/trust/common.trust.data.mapper';
+import { ApplicationData } from '../model/application.model';
+import * as PageModel from '../model/trust.page.model';
+import { TrustKey } from '../model/trust.model';
+
+const INDIVIDUAL_BO_TEXTS = {
+  title: 'Tell us about the individual',
+};
+
+const get = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    const trustId = req.params[config.ROUTE_PARAM_TRUST_ID];
+    const trustData: PageModel.CommonTrustData = CommonTrustDataMapper.mapCommonTrustDataToPage(
+      appData[TrustKey]?.find(trust => trust.trust_id === trustId),
+    );
+
+    const templateName = config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_PAGE;
+
+    return res.render(
+      templateName,
```

Comment:
please move this to getPageProperties function
|please look trust.involved.controller.ts

Reply:
getPageProperties added